packages/ndla-ui/src/NDLAFilm/SlideshowIndicator.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2019-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import BEMHelper from 'react-bem-helper';
+import { IMovie } from './shapes';
+
+interface Props {
+  slideshow: Array<IMovie>;
+  activeSlide: number;
+  gotoSlide: Function;
+}
+
+const classes = new BEMHelper({
+  name: 'film-slideshow',
+  prefix: 'c-',
+});
+
+const SlideshowIndicator: React.FC<Props> = ({ slideshow, activeSlide, gotoSlide }) => {
+  return (
+    <div {...classes('indicator-wrapper')}>
+      {slideshow.map((slide, index) => (
+        <button
+          key={`indicator_${index}`} // eslint-disable-line react/no-array-index-key
```

Comment:
Eslint-kommentaren trengs ikkje lenger.
```suggestion
          key={`indicator_${index}`}
```